systemtest/src/main/java/io/strimzi/systemtest/utils/specific/JmxUtils.java

```diff
@@ -82,4 +82,19 @@ public static String collectJmxMetricsWithWait(String namespace, String serviceN
 
         return result[0];
     }
+
+    public static String collectJmxTransMetricsWithWait(String namespace, String metricName, String jmxTransName) {
+        String jmxTransPodName  = kubeClient(namespace).listPodsByPrefixInName(jmxTransName).get(0).getMetadata().getName();
+
+        String[] result = {""};
+
+        TestUtils.waitFor("JmxTrans metric will be present: " + metricName, Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
+            () -> {
+                result[0] = kubeClient().logs(jmxTransPodName);
```

Comment:
You should ideally go with `kubeClient().logsInSpecificNamespace(namespace, jmxTransPodName);` to prevent any other race conditions.